app/components/drawer_layout_polyfill.js

```diff
@@ -0,0 +1,401 @@
+import React, { Component } from 'react';
+import {
+    Animated,
+    Dimensions,
+    Keyboard,
+    PanResponder,
+    StyleSheet,
+    TouchableWithoutFeedback,
+    View,
+    I18nManager,
+} from 'react-native';
+
+const MIN_SWIPE_DISTANCE = 3;
+const DEVICE_WIDTH = parseFloat(Dimensions.get('window').width);
+const THRESHOLD = DEVICE_WIDTH / 2;
+const VX_MAX = 0.1;
+
+const IDLE = 'Idle';
+const DRAGGING = 'Dragging';
+const SETTLING = 'Settling';
+
+export default class DrawerLayout extends Component {
+    /*static propTypes = {
```

Comment:
why are these commented? can this be removed?

Reply:
it's commented out because the file was initially written in flow and I just wanted it to compile